docs/docs/guides/template-backup-script.md

```diff
@@ -39,6 +23,16 @@ BACKUP_PATH="/path/to/local/backup/directory"
 REMOTE_HOST="remote_host@IP"
 REMOTE_BACKUP_PATH="/path/to/remote/backup/directory"
 
+### init Borg setup
+if [ ! -d "$UPLOAD_LOCATION/database-backup" ]; then
+    mkdir "$UPLOAD_LOCATION/database-backup"
+fi
+if [ ! -d "$BACKUP_PATH/immich-borg" ]; then
+    borg init --encryption=none "$BACKUP_PATH/immich-borg"
+fi
+if ! ssh -o StrictHostKeyChecking=no -oConnectTimeout=10 "$REMOTE_HOST" "test -d $REMOTE_BACKUP_PATH/immich-borg"
```

Comment:
This command will fail if REMOTE_BACKUP_PATH contains a space, the rest of the script currently would support this. In addition, testing for `! ssh` means that if the SSH command itself fails, it will still attempt to run `borg init`.